src/open_inwoner/plans/views.py

```diff
@@ -39,12 +40,14 @@ def dispatch(self, request, *args, **kwargs):
 
 class BasePlanFilter:
     """
-    This will filter the plans according to the user's selection.
+    This will filter the plans according to the user's selection or query.
     """
 
-    def get_filtered_plans(self, plans):
-        plan_contacts = self.request.GET.get("plan_contacts")
-        status = self.request.GET.get("status")
+    def get_filtered_plans(self, plans, available_contacts):
+        args = self.request.GET
```

Comment:
`args` is not a good general use variable name as it is strongly associated with variable function arguments (eg `args` & `kwargs`). For query parameters `params` is a good choice. 

(also I'm not 100% convinced you'd need to alias it here and below but heh)